src/app/(proper_react)/(onboarding)/user/welcome/Onboarding.test.tsx

```diff
@@ -0,0 +1,40 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { it, expect } from "@jest/globals";
+import { render, screen } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+import { composeStory } from "@storybook/react";
+import { axe } from "jest-axe";
+import Meta, { Onboarding } from "./Onboarding.stories";
+
+it("passes the axe accessibility test suite on step 1", async () => {
+  const ComposedOnboarding = composeStory(Onboarding, Meta);
```

Comment:
Look, I'm reusing the Storybook setup! Also, see below on how to run the axe test suite on a component. I've already set it up for step two in the onboarding as well.